Securancy.Labs.KvkApi.Tests/BaseTests.cs

```diff
@@ -4,7 +4,7 @@ public abstract class BaseTests
 {
     protected readonly KvkApiClientConfig Config = new()
     {
-        Endpoint = "https://api.securancy.io/services/kvk",
+        Endpoint = "https://api.securancy.io/services/kvk/v3",
```

Comment:
I assume we need to update this endpoint in consumer apps as well?

Also, why is it v3 if the zoeken api is only v2?

Also also, why is there just a hardcoded api key in source control? :thinking:

Reply:
> I assume we need to update this endpoint in consumer apps as well?

The endpoint does need to be updated in consumer apps, along with the package version.
Reason: I've versioned the API in APIM because the new v2/zoeken endpoint would have conflicting model names with v1/zoeken, causing a mess in the definitions.

> Also, why is it v3 if the zoeken api is only v2?

The version v3 is the new version of our API in APIM, v2 is the prefix KVK requires for the zoeken endpoint.

> Also also, why is there just a hardcoded api key in source control? 🤔

You tell me 😄 I'll remove it 👍
![Screenshot 2024-07-16 at 09 02 18@2x](https://github.com/user-attachments/assets/de359db0-7065-479d-90f8-4da808bf0e26)

Reply:
Hmm, we should definitely rotate the api keys in the apps, just to be sure

Reply:
I've replaced the hardcoded key with an appsettings implementation.
I will rotate the api keys when I update the package in the consumer apps.

Reply:
Thanks, do we have multiple api keys to avoid any outages?

Reply:
Of course.